cmd/manager/main.go

```diff
@@ -83,7 +83,7 @@ func main() {
 		"Indicate if the operator is running on an OpenShift cluster.")
 
 	// NOTE: KEPLER_IMAGE can be set as env or flag, flag takes precedence over env
-	keplerImage := env("KEPLER_IMAGE", exporter.StableImage)
+	keplerImage := env("RELATED_IMAGE_KEPLER", exporter.StableImage)
```

Comment:
is it possible to we use `KEPLER_IMAGE` itself? If it should be `RELATED_IMAGE_KEPLER` then the comment above has to change as well

Reply:
> is it possible to we use `KEPLER_IMAGE` itself? If it should be `RELATED_IMAGE_KEPLER` then the comment above has to change as well

`RELATED_IMAGE_`  should be the way to refer to operand images in CSV
https://docs.openshift.com/container-platform/4.13/operators/operator_sdk/osdk-generating-csvs.html#olm-enabling-operator-for-restricted-network_osdk-generating-csvs

In the downstream CSV, this is a requirement, and a `relatedImages` section is automatically added in the process. just like we have got it generated here.

Reply:
will change the comment